demos/simple-menu.html

```diff
@@ -17,11 +17,13 @@
 <html>
```

Comment:
This demo still doesn't technically show the menu opening from the top-left and top-right. It positions the buttons there, but it's so far down on the page that the menu opens upward rather than downward. My suggestion would be to add more height to the button's container such that when positioned in the top-left or top-right positions, the menu opens from top to bottom.